### PR TITLE
Banner notifications and customer mass actions

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds banner notification and customer mass action extension points.

## Affected pages

- ...

## Links to Magento Open Source code

- ...

### CODE REVIEW

File: src/pages/admin-ui-sdk/release-notes.md [hunk 1]
```diff
@@ -12,11 +12,15 @@ keywords:
...
 
-March 21, 2024
+June 26, 2024
 
 ### Enhancements
 
-* Added the [`customer grid columns` extension point](./extension-points/customer/grid-columns.md).  <!--- CEXT-2867  -->
+* Added the following extension points:
+
+  * [`banner notification](./extension-points/banner-notification.md)
```
Comment (jhadobe) [inline src/pages/admin-ui-sdk/release-notes.md:21]:
```suggestion
  * [`banner notification](./extension-points/banner-notification.md).
```

File: src/pages/admin-ui-sdk/extension-points/banner-notification.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+title: bannerNotification
+description: Create a banner notification mass action in the Adobe Commerce Admin.
+keywords:
+  - App Builder
+  - Extensibility
+---
+
+# bannerNotification
+
+The `bannerNotification` extension point defines the contents of banner notifications that are displayed when a customer, order, or product mass action is attempted.
```
Comment (jhadobe) [inline src/pages/admin-ui-sdk/extension-points/banner-notification.md:11]:
```suggestion
The `bannerNotification` extension point defines the contents of banner notifications that display when attempting a customer, order, or product mass action.
```

**Review by jhadobe — APPROVED** [review]

File: src/pages/admin-ui-sdk/extension-points/index.md [hunk 3]
```diff
@@ -10,6 +10,8 @@ keywords:
 
 This section describes how to use existing extension points in any Adobe Developer App Builder application that customizes Adobe Commerce Admin.
 
+The [Adobe Commerce Samples repository](https://github.com/adobe/adobe-commerce-samples/tree/main/admin-ui-sdk/menu/custom-menu) contains samples for different extension points of the Adobe Commerce Admin UI SDK. Use these samples to gain insight on how the Admin SDK injects menus and pages into the Admin.
```
Comment (asalloum5) [inline src/pages/admin-ui-sdk/extension-points/index.md:13]:
```suggestion
The [Adobe Commerce Samples repository](https://github.com/adobe/adobe-commerce-samples/tree/main/admin-ui-sdk) contains samples for different extension points of the Adobe Commerce Admin UI SDK. Use these samples to gain insight on how the Admin SDK injects menus and pages into the Admin.
```

File: src/pages/admin-ui-sdk/index.md [hunk 4]
```diff
@@ -29,3 +29,7 @@ The SDK contains two main components that are necessary for development:
 The following sequence diagram illustrates the authentication process.
 
 ![Sequence diagram](../_images/admin-ui-sequence-diagram.png)
+
+## Code samples
+
+The [Adobe Commerce Samples repository](https://github.com/adobe/adobe-commerce-samples/tree/main/admin-ui-sdk/menu/custom-menu) contains samples for different extension points of the Adobe Commerce Admin UI SDK. Use these samples to gain insight on how the Admin SDK injects menus and pages into the Admin.
```
Comment (asalloum5) [inline src/pages/admin-ui-sdk/index.md:35]:
```suggestion
The [Adobe Commerce Samples repository](https://github.com/adobe/adobe-commerce-samples/tree/main/admin-ui-sdk) contains samples for different extension points of the Adobe Commerce Admin UI SDK. Use these samples to gain insight on how the Admin SDK injects menus and pages into the Admin.
```

File: src/pages/admin-ui-sdk/extension-points/banner-notification.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+title: bannerNotification
+description: Create a banner notification mass action in the Adobe Commerce Admin.
+keywords:
+  - App Builder
+  - Extensibility
+---
+
+# bannerNotification
+
+The `bannerNotification` extension point defines the contents of banner notifications that display when attempting a customer, order, or product mass action.
```
Comment (asalloum5) [inline src/pages/admin-ui-sdk/extension-points/banner-notification.md:11]: bannerNotification works also for order view button. I updated the wiki with an example. 
https://wiki.corp.adobe.com/display/DMSArchitecture/bannerNotification

File: src/pages/admin-ui-sdk/extension-points/banner-notification.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+`customer.successMessage` | No | The success message to display when mass action is successful. A default message is displayed if this parameter is not defined.
+`customer.errorMessage` | No | The error message to display when mass action is successful. A default message is displayed if this parameter is not defined.
+`order.actionId` | string | Yes | The `actionId` of an [order mass action](./order/mass-action.md).
+`order.successMessage` | No | The success message to display when mass action is successful. A default message is displayed if this parameter is not defined.
+`order.errorMessage` | No | The error message to display when mass action is successful. A default message is displayed if this parameter is not defined.
+`product.actionId` | Yes | The `actionId` of a [product mass action](./product/mass-action.md).
+`product.successMessage` | No | The success message to display when mass action is successful. A default message is displayed if this parameter is not defined.
+`product.errorMessage` | No | The error message to display when mass action is successful. A default message is displayed if this parameter is not defined.
```
Comment (asalloum5) [inline src/pages/admin-ui-sdk/extension-points/banner-notification.md:60]:
```suggestion
`product.errorMessage` | No | The error message to display when mass action fails. A default message is displayed if this parameter is not defined.
```

File: src/pages/admin-ui-sdk/extension-points/banner-notification.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+| Field | Type | Required | Description |
+| --- | --- | --- | --- |
+`customer.actionId` | Yes | The `actionId` of a customer mass action.
+where banner notification will be customized. It should be the same one that is defined in the customer mass action extension point.
+`customer.successMessage` | No | The success message to display when mass action is successful. A default message is displayed if this parameter is not defined.
+`customer.errorMessage` | No | The error message to display when mass action is successful. A default message is displayed if this parameter is not defined.
+`order.actionId` | string | Yes | The `actionId` of an [order mass action](./order/mass-action.md).
+`order.successMessage` | No | The success message to display when mass action is successful. A default message is displayed if this parameter is not defined.
+`order.errorMessage` | No | The error message to display when mass action is successful. A default message is displayed if this parameter is not defined.
```
Comment (asalloum5) [inline src/pages/admin-ui-sdk/extension-points/banner-notification.md:57]:
```suggestion
`order.errorMessage` | No | The error message to display when mass action fails. A default message is displayed if this parameter is not defined.
```

File: src/pages/admin-ui-sdk/extension-points/banner-notification.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+```
+
+## Parameters
+
+| Field | Type | Required | Description |
+| --- | --- | --- | --- |
+`customer.actionId` | Yes | The `actionId` of a customer mass action.
+where banner notification will be customized. It should be the same one that is defined in the customer mass action extension point.
+`customer.successMessage` | No | The success message to display when mass action is successful. A default message is displayed if this parameter is not defined.
+`customer.errorMessage` | No | The error message to display when mass action is successful. A default message is displayed if this parameter is not defined.
```
Comment (asalloum5) [inline src/pages/admin-ui-sdk/extension-points/banner-notification.md:54]:
```suggestion
`customer.errorMessage` | No | The error message to display when mass action fails. A default message is displayed if this parameter is not defined.
```

**Review by asalloum5 — APPROVED** [review]

**Pull request merged**